htdocs/done.php

```diff
@@ -49,6 +49,8 @@
   echo "es";
 } if ($_SESSION['l'] == "it") {
   echo "it";
+} if ($_SESSION['l'] == "fr") {
+  echo "fr";
 }
 ?>">
 <head>
```

Comment:
Between line 83 and 84 is missing a French redirect. In this way users will be able to alternatively select also French when downloading the pdf.